server/tests/test_file_source_api.py

```diff
@@ -0,0 +1,128 @@
+from datetime import timedelta
+
+import pytest
+from django.urls import reverse
+
+from server.files.models import FileSource
+
+
```

Comment:
We should add some tests for listing file sources here...

Reply:
+1